message_processor.go

```diff
@@ -31,6 +32,8 @@ import (
 	protoio "github.com/gogo/protobuf/io"
 )
 
+const envVarJSONLogs = "SPLUNK_LOGGING_DRIVER_JSON_LOGS"
```

Comment:
all param setup and consts are set up in splunk_logger.go
Follow same methodology for adding this param

Reply:
Following same pattern of defining the constant locally

Reply:
sorry thats not the case @gp510 - move const to splunk_logger.go its where all parameter constants are defined.

Reply:
Now moves to splunk_logger.go